sql/core/src/main/scala/org/apache/spark/sql/execution/ui/ToolTips.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.ui
+
+private[ui] object ToolTips {
+  val SQL_TAB_DURATION =
+    "Difference between start time and close time"
+
+  val SQL_TAB_SUBMITTED =
+    "Job submission time"
+
+  val SQL_TAB_DESCRIPTION =
+    "Description of submitted Job"
```

Comment:
nit: may be, "Description of the SQL query. When click on the description link, it will navigate to query details page."

Reply:
ok this i will change

Reply:
Yes. There can be multiple jobs corresponding to one sql query.

Reply:
ok